e2e-tests/denylist/run-tests.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+source ../util.sh
+
+echo "------------------------------------"
+echo "Cleaning up running Docker containers and volumes, and key material..."
+echo "------------------------------------"
+docker compose down
+if [ $? -ne 0 ]; then
+	echo "ERROR: failed to shut down old containers"
+	exitWithDockerLogs 1
+fi
+
+docker compose rm -f -v
+if [ $? -ne 0 ]; then
+	echo "ERROR: failed to remove old containers"
+	exitWithDockerLogs 1
+fi
+
+echo "------------------------------------"
+echo "Starting Docker containers..."
+echo "------------------------------------"
+docker compose up --wait
+if [ $? -ne 0 ]; then
+	echo "ERROR: failed to start containers"
+	exitWithDockerLogs 1
+fi
+
+sleep 5
+
+# Simply to log this
+openssl version
+
+echo "------------------------------------"
+echo "Connecting (defaults) with allowed cert.."
+echo "------------------------------------"
+for x in $(seq 100); do
```

Comment:
do we need to run all these tests 100 times? If the test is triggered on ever commit it runs ofter enough I'd say